xmtp/src/conversation.rs

```diff
@@ -140,11 +157,29 @@ where
             self.client.account.addr(),
             content_bytes,
             MessageState::Unprocessed as i32,
+            now(),
         )
         .store(&mut self.client.store.conn().unwrap())?;
         Ok(())
     }
 
+    pub async fn list_messages(
+        &self,
+        opts: ListMessagesOptions,
```

Comment:
Can the listMessageOptions struct be passed by ref in this case? would avoid having to create multiple objects for successive queries.

Reply:
That's a good idea